index.html

```diff
@@ -276,6 +276,45 @@ <h3 class="im-qaisar">Glad to see you!</h3>
           </ul>
         </div>
     </section>
+    <section>
+      <div class="contact-me-form-container">
+        <h1 class="contact-me-form-heading">Contact me</h1>
+        <p class="contact-me-form-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it
+        </p>
+        <form  class="contact-me-form">
+          <input
+            class="contact-me-form-text-input"
+            required
+            maxlength="30"
+            name="name"
+            type="text"
+            placeholder="Your good name"
+          />
+          <input
+            class="contact-me-form-text-input"
+            required
+            name="email"
+            type="email"
+            placeholder="Your email address"
+          />
+          <textarea
+            class="contact-me-form-text-input"
+            required
+            maxlength="500"
+            name="message"
+            cols="30"
+            rows="9"
+            placeholder="Write your message here"
+          ></textarea>
+          <button class="contact-me-form-button" type="submit">
+            Get In Touch
+          </button>
```

Comment:
## Styling ✨✨

- [ ] Kindly, it would be better to replicate the given ```colors, size, alignment, and spacing``` of your button as given by the Figma design 😎😎

- [ ] It would be better also to add the button interactions such as **hover, active (when it's pressed), disabled, and enable (default state)** as shown in the Figma template. 🙏 

   Replicating the `styling` from the Figma design will help ensure that the final product looks consistent and polished.  😎😎

### You can check the comparison below 👇 👇

**Your design**
![image](https://github.com/Kaiserabbas/Portfolio/assets/102757126/c34afeda-f7e5-42b6-b077-c8521ed851fd)

**Figma buttons**
![image](https://github.com/Kaiserabbas/Portfolio/assets/102757126/af223246-8b08-4f73-a6f1-b392c54e3a80)